test/Approximations/overapproximate_expmap.jl

```diff
@@ -51,6 +51,27 @@ for N in @tN([Float32, Float64])
         pts = sample(Pex, 10; sampler=sampler)
         @test all(p ∈ Zex for p in pts)
 
+        # overapproximate MatrixZonotopeExp
```

Comment:
I think these tests should be wrapped in a
`@static if isdefined(@__MODULE__, :IntervalMatrices)`
block.